app/models/article.rb

```diff
@@ -22,6 +22,7 @@ def has_friendly_id_slug?
   has_many :subscribers, through: :subscriptions, class_name: "User", source: :user
   has_many :endorsements, class_name: "ArticleEndorsement", counter_cache: true, dependent: :destroy
   has_many :endorsers, through: :endorsements, class_name: "User", source: :user
+  validates :title, presence: true, uniqueness: true, allow_nil: true
```

Comment:
@SkyWDev it's a little weird validate `presence` and `allow_nil` on the same attribute.